cache/edge_lambdas/toggler.js

```diff
@@ -37,10 +64,15 @@ function parseToggleCookies(cookieHeader) {
   return cookies;
 }
 
+function randomFromRange(min, max) {
+  return Math.floor(Math.random() * (max - min + 1) + min);
+}
+
 exports.request = (event, context) => {
   const request = event.Records[0].cf.request;
   const toggleCookies = parseToggleCookies(request.headers.cookie);
 
+  const point = randomFromRange(0, 99);
```

Comment:
should this max @ 29?

Reply:
or are you not intending to assign all the traffic to one of the ranges?

Reply:
The possible range is 0-99.
The tests will be taking up 30% of that, 10% each.

Reply:
👍